test/libsolidity/syntaxTests/types/mapping/library_storage_parameter_with_mapping_2.sol

```diff
@@ -0,0 +1,4 @@
+struct S { mapping(uint => uint)[2] a; }
+library L {
+    function f(S storage s) public {}
+}
```

Comment:
Does this test actually get executed? It's missing the `// ----` and `isoltest` usually fails if I forget it.

Reply:
My understanding is that `isoltest` keeps the `// ----`, if it is there. For instance, if all errors are gone, `isoltest` updates the `.sol` file, adding the trailing `// ----`.

However if a test has no `// ----` and succeeds, `isoltest` (and `soltest`) are happy.

Reply:
Looks like it complains only in semantic tests:
```
Exception during test: /solidity/test/libsolidity/SemanticTest.cpp(101): Throw in function solidity::frontend::test::SemanticTest::SemanticTest(const string&, solidity::langutil::EVMVersion, const std::vector<boost::filesystem::path>&, bool)
Dynamic exception type: boost::wrapexcept<std::runtime_error>
std::exception::what: No tests specified in /projects/solidity/test/libsolidity/semanticTests/enums/constructing_enums_from_ints.sol
```

OK then, if it passes then I guess it's fine.